packages/name-controller/src/providers/etherscan.ts

```diff
@@ -144,6 +147,7 @@ export class EtherscanNameProvider implements NameProvider {
 
       return { responseData };
     } catch (error) {
+      assertIsError(error);
```

Comment:
The update to v4.8.4 seems to have changed the effective type of this error variable for some reason. It was now triggering a lint error in a function that called this one, due to the `@typescript-eslint/no-throw-literal` rule (the error was typed as `{}`, as it was `unknown` and then we confirmed that it was not falsy).

We now ensure it's explicitly typed as an Error, as we would expect.